Core/Sources/Model/LocationUpdate.swift

```diff
@@ -25,13 +25,13 @@ public class EnhancedLocationUpdate {
 
 public class EnhancedLocationUpdateMessage: Codable {
```

Comment:
This is only used in the internal communication between Subscriber and Publisher, right? It's not visible from the outside of our SDKs? I'm asking because I've noticed that it's a `public class` :wink:

Reply:
Thanks for catching that. I left it the way it was originally and didn't think about it. I noticed that many classes which seem to be for internal use are public in other files too. I've removed public from the other classes in the file (`LocationUpdateType`, `EnhancedLocationUpdate`, and `EnhancedLocationUpdateMessage`), but there are many more instances of over-exposure 😅.